crates/core/plugin_sm/src/plugin_manager.rs

```diff
@@ -96,7 +96,10 @@ impl ExternalPlugins {
             default_plugin_type: default_plugin_type.clone(),
             sudo,
         };
-        let () = plugins.load()?;
+        if let Err(e) = plugins.load() {
+            warn!("{:?}: {:?}", e.kind(), &plugins.plugin_dir);
```

Comment:
That warning log statement should give more context, something like:
```suggestion
            warn!("Reading the plugins directory: failed with : {:?}: {:?}", &plugins.plugin_dir, e.kind());
```